### ISSUE TITLE
Unable to pass worksheet to ExcelFile object instantiation?

### ISSUE THREAD

**Issue by Snotpus** [Mikuana/rumydata#51]
It seems that kwargs passed to `ExcelFile` aren't being acknowledged. 
Consider the following code:
`excel_kwargs = {'sheet': 'Sheet5'}`
`ExcelFile(layout, **excel_kwargs).check(r"C:\mytestfile.xlsx")`

This seems to use the default of the active sheet, rather than the specified sheet

**Comment by Mikuana** [comment]
This was an error in the `ExcelFile` context handler. The worksheet was selected by the kwargs sheet name, but the conditional check was based upon the `sheet` param, which was always empty. This has been corrected to remove the sheet parameter and rely on the sheet name passed in the kwargs instead.

```
    def _rows(self, file: Path, sheet=None):
        class Handler:
            def __init__(self, file_path, **kwargs):
                self.file_path = file_path
                self.excel_kwargs = kwargs

            def __enter__(self) -> Iterable:
                from openpyxl import load_workbook
                wb = load_workbook(file)
                sheet_name = self.excel_kwargs.get('sheet')
                ws = wb.get_sheet_by_name(sheet_name) if sheet else wb.active
                return ws.values
```

**Comment by Mikuana** [comment]
Resolved in #52
